mathjax3-ts/output/chtml/Wrappers/scriptbase.ts

```diff
@@ -105,10 +145,10 @@ export class CHTMLscriptbase extends CHTMLWrapper {
      *
      * @param{BBox} bbox   The bounding box of the base element
      * @param{BBox} sbox   The bounding box of the script element
-     * @return{number}     The vertical offset for the script
+     * @return{number[]}   The horizontal and vertical offsets for the script
      */
     protected getOffset(bbox: BBox, sbox: BBox) {
-        return 0;
+        return [0, 0];
```

Comment:
I am wondering if we should have a type for pairs and triples of numbers. They appear a lot and might help finding bugs. number[] could be too general. Just a thought, maybe for the future.

Reply:
OK, we can consider that for the future.